frontend/src/components/pages/connect/CreateConnector.tsx

```diff
@@ -100,11 +99,13 @@ class CreateConnector extends PageComponent {
         if (clusters == null) return null;
 
         return (
-            <motion.div {...animProps} className={styles.motionContainer}>
-                <Card className={styles.wizardView}>
-                    <ConnectorWizard connectClusters={clusters} />
-                </Card>
-            </motion.div>
+            <PageContent>
+                <Section>
+                    <div className={styles.wizardView}>
+                        <ConnectorWizard connectClusters={clusters} />
+                    </div>
+                </Section>
+            </PageContent>
```

Comment:
The card previously had the wizard view style, now the div with the wizard view is nested inside the new component. Are you sure this will display the same as before?
If wizardView has margin for example, nesting might give a different result.

Reply:
Here's the wizard view.  It still works the same.  Section doesn't have 'className' as a prop, so i added the div inside incase you were targeting the class for any reason.

<img width="1345" alt="image" src="https://user-images.githubusercontent.com/523614/192905645-ea113ef1-53a7-4171-b046-679db4b1fb1a.png">